docs/resources/oceancd_rollout_spec.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: "spotinst"
+page_title: "Spotinst: oceancd_rollout_spec"
+subcategory: "OceanCD"
+description: |-
+  Provides a Spotinst OceanCD Rollout Spec resource.
+---
+
+# spotinst\_oceancd\_rollout\_spec
+
+Manages a Spotinst OceanCD Rollout Spec resource.
+
+## Example Usage
+
+```hcl
+resource "spotinst_oceancd_rollout_spec" "example" {
+  
+  rollout_spec_name  = "test-rollout_spec"
+  
+  // --- Failure Policy ----------------------------------------------------------------
+  failure_policy {
+    action = "abort"
+  }
+  
+  //------------------------------------------------------------------------------------
+
+
+  // --- Spot Deployment ----------------------------------------------------------------
+ 
+  spot_deployment {
+    cluster_id = "Test-Cluster-Id"
```

Comment:
modify the attribute names according to the consts file

Reply:
modified